src/viewer.js

```diff
@@ -4438,15 +4439,6 @@ class VolumeImageViewer {
       source.setLoader(loader)
     }
 
-    const view = this[_map].getView()
-    const currentZoomLevel = view.getZoom()
```

Comment:
@pedrokohler please add a new option to enable this if people need it.

Reply:
@igoroctaviano do you mean a config option?

@fedorov @igoroctaviano  do you think we need to add an option to allow people to enable the automatic zoom when the segmentation is activated? Frankly, I don't see the point, since it was just annoying and didn't seem very helpful. But maybe you know of an use case.

Reply:
I think, and if I remember correctly discussing this with Markus and/or Igor, automatic zoom might have been motivated by performance considerations. If segmentation is not stored as a pyramid, starting at the low-res level may trigger resampling and might be time-consuming.

I agree with Igor that it might be helpful to add this as a configuration option.

@CPBridge @pieper what do you think?

Reply:
I don't have a strong feeling - maybe if we could compare the two options head-to-head it would make the tradeoffs more clear.

Reply:
Since it's an open question, I'll just add a parameter to the function so it's configurable, since it's gonna be faster than finding datasets to test this.

If it makes sense to anyone, they can set the parameter to true and enable the auto zoom.